### PR TITLE
Add io.github.verticalsync.sunroof

### PR DESCRIPTION
**Pull request by verysillycat**
### Please confirm your submission meets all the criteria

Sunroof is a community project centered around a lightweight, fast and fully customisable alternative to Discord. The perfect fitting for Vesktop, another open-source Discord client, Sunroof takes everything good in Vesktop and adds extra plugins & some improvements uniquely catered for Discord.

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by hfiguiere** [comment]
This is missing:

> Please describe your application briefly.

File: me.verticalsync.sunroof.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      - install -D squashfs-root/sunroof.desktop /app/share/applications/me.verticalsync.sunroof.desktop
+      - install -Dm755 startsunroof /app/bin/startsunroof
+      - mv squashfs-root /app/bin/sunroof
+
+    sources:
+      - type: file
+        path: startsunroof
+
+      - type: file
+        url: https://raw.githubusercontent.com/verticalsync/Sunroof/main/meta/me.verticalsync.Sunroof.metainfo.xml
```
Comment (hfiguiere) [inline me.verticalsync.sunroof.yml:48]: this should NOT reference a branch.

File: me.verticalsync.sunroof.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        sha512: 88f2ab45cf749135ee1fd0c95cff3244f56b66b5b28c21e45adbd699d3c4e24b8da800433624bab2529af8b4ab088d33b5b1d0d0657282aafdc9829894e85622
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/verticalsync/Sunroof/contents/meta/me.verticalsync.Sunroof.metainfo.xml
+          version-query: .sha
+          url-query: .download_url
+
+      - type: file
+        url: https://github.com/verticalsync/Sunroof/releases/latest/download/Sunroof-1.6.0.AppImage
```
Comment (hfiguiere) [inline me.verticalsync.sunroof.yml:57]: this should not reference "latest" but the actual version. Same below.

File: me.verticalsync.sunroof.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+app-id: me.verticalsync.sunroof
```
Comment (hfiguiere) [inline me.verticalsync.sunroof.yml:1]: `verticalsync.me` doesn't exist it seems. Albeit you did check

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

File: me.verticalsync.sunroof.yml (hunk 1 above)
Comment (hfiguiere) [reply me.verticalsync.sunroof.yml]: this wasn't resolved.

**Comment by hfiguiere** [comment]
bot, build io.github.verticalsync.sunroof

File: io.github.verticalsync.sunroof.yml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      - install -D squashfs-root/sunroof.desktop /app/share/applications/io.github.verticalsync.sunroof.desktop
+      - install -Dm755 startsunroof /app/bin/startsunroof
+      - mv squashfs-root /app/bin/sunroof
+
+    sources:
+      - type: file
+        path: startsunroof
+
+      - type: file
+        url: https://raw.githubusercontent.com/verticalsync/Sunroof/master/meta/me.verticalsync.Sunroof.metainfo.xml
```
Comment (hfiguiere) [inline io.github.verticalsync.sunroof.yml:48]: the name MUST match the app-id. It's case sensitive. Also even with the wrong name it is name installed.

File: me.verticalsync.sunroof.yml (hunk 1 above)
Comment (hfiguiere) [reply me.verticalsync.sunroof.yml]: No it is not required. You can put a tag or a commit instead. Even with the UI you can do it.

**Comment by verysillycat** [comment]
Seems like Github doesn't like when someone ranames branches

**Comment by bbhtt** [comment]
bot, build io.github.verticalsync.sunroof

**Pull request closed without merging**